Netcoreapp2.0move

sbaid opened the pull request.

sbaid committed:
build vstest.console.dll as netcoreapp2.0

sbaid committed:
test script changes

msftclas commented:
@sbaid,
Thanks for your contribution as a Microsoft full-time employee or intern. You do not need to sign a CLA.
_Thanks,
Microsoft Pull Request Bot_

sbaid committed:
Fixing acceptance tests - core runner frameowork as netcoreapp2.0

sbaid commented:
@dotnet-bot test debug please

sbaid commented:
@dotnet-bot test please

sbaid commented:
@dotnet-bot help

sbaid commented:
@dotnet-bot build Windows_NT debug please

smadala committed:
stream test run to console

smadala committed:
Move TestPlatform.CLI artifacts to netcoreapp2.0 folder

smadala committed:
Ignore failing platform tests

smadala committed:
Merge branch 'master' into netcoreapp2.0move

smadala committed:
Merge branch 'master' into netcoreapp2.0move

smadala committed:
Merge branch 'master' of https://github.com/Microsoft/vstest into netcoreapp2.0move

Commit:
Install netcoreapp1.1 shared runtime.

smadala committed:
Match System.Xml.XPath.XmlDocument and DependencyModel version with dotnet cli

smadala committed:
Merge branch 'netcoreapp2.0move' of https://github.com/Microsoft/vstest into netcoreapp2.0move

Commit:
Install netcoreapp 1.1.1 shared runtime for acceptance tests targeting netcoreapp1.1.
Use async stdout, stderr listeners for IntegrationTestBase to ensure there are no deadlocks
in the child processes created for acceptance tests.

codito committed:
Merge branch 'master' into netcoreapp2.0move

smadala committed:
Revert nuspec2 files

smadala committed:
Merge branch 'netcoreapp2.0move' of https://github.com/Microsoft/vstest into netcoreapp2.0move

smadala committed:
uncomment build steps

Commit:
Move DataCollectors to netcoreapp2.0. Use the datacollector for the runner framework
for acceptance tests.

Commit:
Enable vstest.console and translation layer unit tests.
Fix path for datacollection acceptance tests sample data collector.

codito committed:
Merge branch 'master' into netcoreapp2.0move

codito commented:
@dotnet-bot test this please

Commit:
Add target fx as a parameter to run specific tests in test.ps1.
Call WaitForExit twice in IntegrationTestBase to ensure streams are flushed.
Ignore a failing test in net46 temporarily.

smadala committed:
Make error null to handle JSON serialize exception

smadala committed:
Merge branch 'netcoreapp2.0move' of https://github.com/Microsoft/vstest into netcoreapp2.0move

Commit:
Use netcoreapp2.0 as the base directory for core clr package.

codito committed:
Skip target framework only if not null.
